controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -907,7 +958,7 @@ func validateControlPlaneSpec(ocmClient rosa.OCMClient, rosaScope *scope.ROSACon
 	return "", nil
 }
 
-func buildOCMClusterSpec(controlPlaneSpec rosacontrolplanev1.RosaControlPlaneSpec, creator *rosaaws.Creator) (ocm.Spec, error) {
+func buildOCMClusterSpec(controlPlaneSpec rosacontrolplanev1.RosaControlPlaneSpec, roleConfig *expinfrav1.ROSARoleConfig, creator *rosaaws.Creator) (ocm.Spec, error) {
```

Comment:
ExternalAuth need to be defined as well in the ROSARoleConfig and assigned to the cluster spec similar to [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/controlplane/rosa/controllers/rosacontrolplane_controller.go#L946)

Reply:
We have discussed this on meeting and we dont need to include `ExternalAuth` in RosaRoleConfig.